src/runtime/runtime_internal.h

```diff
@@ -186,6 +186,8 @@ struct halide_pseudostack_slot_t {
 WEAK void halide_use_jit_module();
 WEAK void halide_release_jit_module();
 
+WEAK int halide_enable_amx();
```

Comment:
If we expect users to call this, I think you want the declaration in HalideRuntime.h instead.

Reply:
I'm having some trouble manually calling the function from outside of a kernel. I think it's because the runtime is only linked into the kernel's object file, I'm guessing I'd have to explicitly link the runtime into the hosting binary as well?